app/views/govuk_publishing_components/components/_layout_main.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="gem-c-layout-main govuk-width-container">
```

Comment:
Does this intend to be the semantic area that the skip-link links too?

In the regular page template we have:

```html
  <div class="govuk-width-container">
    <!-- breadcrumbs, backlink, phasebanner goes here-->
    <main class="govuk-main-wrapper " id="main-content" role="main">
      <h1 class="govuk-heading-xl">Default page template</h1>
    </main>
  </div>
```

https://design-system.service.gov.uk/styles/page-template/

Reply:
I have the main container in the skip link PR coming up. But can be added here too.

Reply:
Hmm, I'm not sure. Let's make this the responsibility of the app at first, after that we'll see how to abstract it into this component.